src/app/shell/info/rules/rules.component.html

```diff
@@ -1,20 +1,22 @@
-<div *ngIf="(platformRules$ | async)?.companyInformationItems.length; else noInfo">
-  <div class="info-content">
-    <div>
-      <h2 class="info-title">{{ (platformRules$ | async)?.title }}</h2>
-    </div>
-    <div class="info-text">
-      <div *ngFor="let item of (platformRules$ | async)?.companyInformationItems">
-        <p><strong>{{ item.sectionName }}</strong></p>
-        <p>{{ item.description }}</p>
+<ng-container *ngIf="!(isLoading$ | async)">
```

Comment:
I see this logic is common for all pages, and as I remember there is a wrapper for these components sp let's move it there 
![image](https://user-images.githubusercontent.com/27493959/182631214-af2deefa-91ac-44bc-8165-6187ff757fc0.png)

Reply:
didn’t bring the logic into this component, because I get an endless load. we dispatch in child components like this.store.dispatch(new GetSupportInformation()). so I left it as it was